src/rollup/rollup.ts

```diff
@@ -44,7 +44,8 @@ export async function rollup(
   };
 
   try {
-    const bundle = await _rollup(options);
+    // deno-lint-ignore no-explicit-any
+    const bundle = await _rollup(options as any);
```

Comment:
Guessing seeing a type mismatch? Check out some of the extended types around https://github.com/cmorten/deno-rollup/blob/main/src/rollup/mod.ts#L165, suspect may have to add the new `sanitizeFileName` or sim. somewhere, shouldn’t need to cast as `any`.

Reply:
i found the problem, now this is fixed